torch/csrc/jit/export.cpp

```diff
@@ -73,8 +73,7 @@ void validateBlock(
       bool is_aten_enabled = operator_export_type ==
               onnx_torch::OperatorExportTypes::ONNX_ATEN_FALLBACK ||
           operator_export_type == onnx_torch::OperatorExportTypes::ONNX_ATEN;
-      if (!node->kind().is_onnx() && !node->kind().is_caffe2() &&
-          !is_aten_enabled && !node->mustBeNone()) {
+      if (node->kind().is_aten() && !is_aten_enabled) {
```

Comment:
The if statement here was blocking export of operators with custom domain and I don't think this is necessary.
We should enable export of ops with domains other thatn caffe2, onnx, and aten. However if aten export type is disabled, aten ops should fail to export.

Reply:
should we still include `!node->mustBeNone()`?